### PR TITLE
add more description to readme

### PR DESCRIPTION
**Pull request by kswang2400**

### CODE REVIEW

File: ibr/README.md [hunk 1]
```diff
@@ -3,7 +3,11 @@ IBR implements single file format that can be adapted to multiple use cases in a
...
 
-IBR has no opinion of data requirements. If a developer wants to use IBR as an asset tracking tool, they can utilize the Objects message. If they later want to map asset locations to a single floor, that is possible by including a Layers message while updating the Objects messages with location data. If they want to track assets with relation to specific space classes, then they can add that data at a later stage without making a breaking change.
+IBR has no opinion of data requirements and is meant to be very lightweight. If a developer wants to use IBR as an asset tracking tool, they can utilize the Objects message. If they later want to map asset locations to a single floor, that is possible by including a Layers message while updating the Objects messages with location data. If they want to track assets with relation to specific space classes, then they can add that data at a later stage without making a breaking change.
```
Comment (charbull) [inline ibr/README.md:6]: representation?

Comment (charbull) [inline ibr/README.md:6]: Layers --> lower case or put it between ` `?

File: ibr/README.md [hunk 2]
```diff
@@ -3,7 +3,11 @@ IBR implements single file format that can be adapted to multiple use cases in a
...
+
+IBR comes with a rendering library built on THREE.js that can be used to easily create a custom UI to visualize and edit the compact data. This allows developers to build custom features and have more control over their tooling.
```
Comment (charbull) [inline ibr/README.md:8]: `THREE.js`?

File: ibr/README.md [hunk 3]
```diff
@@ -3,7 +3,11 @@ IBR implements single file format that can be adapted to multiple use cases in a
...
+
+IBR wants to democratize usage of CAD technology. A lot of existing CAD tools require many hours of training (AutoCAD/Revit/BIM) and is not very accessible to most faciltiies operators who just want to minor changes to a small, specific layer. IBR allows anyone with a browser and IBR file to visualize, edit, and save building data without having to learn one of the existing industry standard tools.
```
Comment (charbull) [inline ibr/README.md:10]: seeks?

File: ibr/README.md (hunk 1 above)
Comment (charbull) [inline ibr/README.md:6]: what is the Objects message? specific to our design? if the case refer to it?

Comment (charbull) [inline ibr/README.md:6]: What is a layers message? reference ? link?

Comment (charbull) [inline ibr/README.md:6]: re-word the ifs?
* Object messages (link?) can be used to represent asset representation?
* Layers messages (link?) can be used to ...?

**Comment by tasodorff** [comment]
@kswang2400 @charbull have you sorted out your rewordings? please confirm if this still needs to be adjusted before merging.

**Pull request closed without merging**